src/main/javascript/src/views/assignment/MultipleAttemptsSetting.vue

```diff
@@ -0,0 +1,193 @@
+          hide-details
+        ></v-checkbox>
+      </v-card-title>
+      <v-card-text v-if="allowMultipleAttempts" class="text--primary mx-0 px-5">
+        <div class="d-flex flex-wrap align-baseline">
+          <v-radio-group
+              v-model="allowInfiniteSubmissions"
+              column
+            >
+              <v-radio
+                label="blue"
+                color="blue"
+                class="mb-5"
+                :value="false"
+              >
+                <template v-slot:label>
+                    <div>A student is allowed up to <v-text-field
+                            outlined
+                            dense
+                            single-line
+                            hide-details
+                            :required="!allowInfiniteSubmissions"
+                            type="number"
+                            class="d-inline-block"
+                            :disabled="allowInfiniteSubmissions"
+                            v-model="numOfSubmissions"
+                            style="width:100px"
+                        ></v-text-field> attempts
+                    </div>
+                </template>
+              </v-radio>
+              <v-radio
+                label="A student is allowed an infinite number of attempts"
+                color="blue"
+                :value="true"
+              ></v-radio>
+          </v-radio-group>
+        </div>
+        <div class="mb-4"><v-divider class="mx-0 px-0"></v-divider></div>
+        
+        <div class="d-flex flex-column flex-wrap align-baseline">
+            <div class="mb-5">Minimum time between submissions: <v-text-field
+                    outlined
+                    dense
+                    single-line
+                    hide-details
+                    type="number"
+                    class="d-inline-block"
+                    v-model="hoursBetweenSubmissions"
+                    style="width:100px"
+                ></v-text-field> hours
+            </div>
+            <div class="mb-5">Keep the <v-select
+                :items="cumulativeOptions"
+                outlined
+                single-line
+                hide-details
+                dense
+                v-model="multipleSubmissionScoringScheme"
+                class="d-inline-block"
+                item-text="label"
+                item-value="value"
+                ></v-select> treatment score
+            </div>
+            <div class="mb-5" v-if="multipleSubmissionScoringScheme === 'CUMULATIVE'">Proportion earned on first attempt: <v-text-field
+                    outlined
+                    dense
+                    single-line
+                    hide-details
+                    type="number"
+                    class="d-inline-block"
+                    v-model="cumulativeScoringInitialPercentage"
+                    style="width:100px"
+                ></v-text-field> %
```

Comment:
I've pushed a fix for this, but I believe the first percent (since it says "remaining") should be (100 - provided), and I think I need to talk to Harmony about text for if the user puts "2" in the number of submissions, making the text slightly odd since it would be "The remaining 50% will be distributed evenly among the other 1 attempts (50% per attempt)."

Reply:
I have updated the text according to instruction from Harmony